go.mod

```diff
@@ -138,6 +138,7 @@ require (
 	github.com/tidwall/gjson v1.14.4 // indirect
 	github.com/tidwall/match v1.1.1 // indirect
 	github.com/tidwall/pretty v1.2.0 // indirect
+	github.com/whilp/git-urls v0.0.0-20191001220047-6db9661140c0 // indirect
```

Comment:
I was wondering if we should use a `replace` rule to get rid of all the instances in `github.com/whilp/git-urls` across our dependencies.

Reply:
Yeah good idea, come to think of it @BKirov that's the only way to be sure we clear up your image scanners.

Reply:
fixed , can you please approve

Reply:
Hm. The vunlerable version is still showing up as an indirect dependency. Do you need two replaces?

```
github.com/whilp/git-urls v0.0.0-20191001220047-6db9661140c0 // indirect
```